msteams-platform/messaging-extensions/how-to/micro-capabilities-for-website-links.md

```diff
@@ -0,0 +1,204 @@
+
+# [Local business](#tab/local-buisness)
+
+|Property |Description  |
+|---------|---------|
+|`@type`     |  Local business       |
+|image    | URL of the image of for the business.        |
+|name    |  Name of the business.       |
+|aggregateRating.ratingValue     | Average Rating of the business.        |
+|address      | Physical address of the business.        |
+|geo.latitude     | Latitude of the business location.        |
+|geo.longitude      | Longitude of the business location.        |
+|aggregateRating.reviewCount      | Average number of reviews for the business.        |
+|aggregateRating.ratingCount       | Average number of ratings for the business.        |
+
+Example of the unfurling experience for the local business type:
+
+:::image type="content" source="../../assets/images/messaging-extension/micro-capabilities-template-local-business.png" alt-text="Screenshot shows the unfurling experience of the local business template in Microsoft Teams.":::
+
+# [Education](#tab/education)
+
+|Property |Description  |
+|---------|---------|
+|`@type`     |   Course      |
+|image    | URL of the image of for the course.        |
+|name    |  Name of the course.       |
+|priceRange    |  Price of the course.       |
+|aggregateRating.ratingValue      |  Average rating of the course.       |
+|description    |  Summary about the course.       |
+|url     |  URL of the course website.       |
+|aggregateRating.reviewCount      | Average number of reviews for the course.        |
+|aggregateRating.ratingCount      | Average number of ratings for the course.        |
+
+Example of the unfurling experience for the course type:
+
+:::image type="content" source="../../assets/images/messaging-extension/micro-capabilities-template-course.png" alt-text="Screenshot shows the unfurling experience of the course template in Microsoft Teams.":::
+
+# [Restaurant](#tab/restaurant)
+
+|Property |Description  |
+|---------|---------|
+|`@type`     |  Restaurant       |
+|image    | URL of the image of for the restaurant.        |
+|name    |  Name of the author.       |
+|priceRange      |  Price range of the restaurant.       |
+|aggregateRating.ratingValue       | Average Rating of the restaurant.        |
+|address     |  Physical address of the restaurant.       |
+|url     | Link to the restaurant's website.        |
+|geo.latitude     | Latitude of the restaurant location.       |
+|geo.longitude      |  Longitude of the restaurant location.       |
+|aggregateRating.reviewCount      | Average number of reviews for the restaurant.        |
+|aggregateRating.ratingCount      | Average number of ratings for the restaurant.        |
+
+Example of the unfurling experience for the restaurant type:
+:::image type="content" source="../../assets/images/messaging-extension/micro-capabilities-template-restaurant.png" alt-text="Screenshot shows the unfurling experience of the restaurant template in Microsoft Teams.":::
```

Comment:
The 'Direction' button doesn't use the `streetAddress` property. It uses `geo.latitude` and `geo.longitude`.

Reply:
The latest schema image shared 

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/c1825957-a181-42e1-abe9-6aa086dc27e8)